src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -578,8 +578,12 @@ private TypeSymbol BindNamedAttributeArgumentType(AttributeArgumentSyntax namedA
 
                 if (parameter.IsParams && parameter.Type.IsSZArray() && i + 1 == parameterCount)
                 {
-                    reorderedArgument = GetParamArrayArgument(parameter, constructorArgsArray, argumentsCount, argsConsumedCount, this.Conversions);
-                    sourceIndices = sourceIndices ?? CreateSourceIndicesArray(i, parameterCount);
+                    reorderedArgument = GetParamArrayArgument(parameter, constructorArgsArray, constructorArgumentNamesOpt, argumentsCount,
+                        argsConsumedCount, this.Conversions, out bool foundNamed);
+                    if (!foundNamed)
```

Comment:
>if (!foundNamed) [](start = 20, length = 16)

It doesn't feel right that we don't allocate the ```sourceIndices``` for the case when the argument is named, but the form is not expanded.